public/js/populateDropdowns.js

```diff
@@ -27,7 +27,7 @@ let outputCohorts = async () => {
         cohortOptions += `value="${item.id}">${item.date}</option>`;
     });
 
-    element.innerHTML = cohortOptions;
+    element.innerHTML += cohortOptions;
 };
 
 let outputHearAbout = async () => {
```

Comment:
can this be a const? sorry just can't see it being re-assigned unless this is the reassignment? Just ignore if i'm wrong.

Reply:
This isn't our code so no clue.